mava/configs/env/rware.yaml

```
@@ -1,7 +1,7 @@
 # ---Environment Configs---
 defaults:
   - _self_
-  - scenario: tiny-2ag # [tiny-2ag, tiny-4ag, tiny-4ag-easy, small-4ag]
+  - scenario: small-4ag # [tiny-2ag, tiny-4ag, tiny-4ag-easy, small-4ag]
```

Spam: revert before merge
```suggestion
  - scenario: tiny-2ag # [tiny-2ag, tiny-4ag, tiny-4ag-easy, small-4ag]
```

---

Done.